Modules/Quotas/src/Quotas.Infrastructure/Persistence/Repository/RelationshipsRepository.cs

```diff
@@ -19,7 +19,8 @@ public async Task<uint> Count(string createdBy, DateTime createdAtFrom, DateTime
         var relationshipsCount = await _relationshipsReadonly
             .CreatedInInterval(createdAtFrom, createdAtTo)
             .Where(r => r.Status == RelationshipStatus.Pending && r.From == createdBy ||
-                r.Status == RelationshipStatus.Active && (r.From == createdBy || r.To == createdBy))
+                r.Status == RelationshipStatus.Active && (r.From == createdBy || r.To == createdBy) ||
+                r.Status == RelationshipStatus.Terminated && r.AuditLog.Last().Reason == RelationshipAuditLogEntryReason.Reactivation)
```

Comment:
I'm not sure if `.Last()` can be translated to SQL, or if it would require to load the audit log into memory. To check that, run a manual test and check the log afterwards. If it doesn't contain a warning, we're good.

Reply:
Changes made, we're good. No warnings.